src/odbc_connection.cpp

```diff
@@ -1845,8 +1862,8 @@ class CallProcedureAsyncWorker : public ODBCAsyncWorker {
 #else
       // Note: On Windows, %s and %S change their behavior depending on whether
       // it's passed to a printf function or a wprintf function. Since we're passing
-      // narrow strings to a wide function, we need to use %S.
-      swprintf(data->sql, 255, L"{ CALL %S (%S) }", combinedProcedureName, parameterString);
+      // narrow strings to a wide function in the case of parameters, we need to use %S.
+      swprintf(data->sql, 255, L"{ CALL %s (%S) }", combinedProcedureName, parameterString);
```

Comment:
So the SQL buffer is only 255 characters, but the procedure name itself is now 1024?

Reply:
Well when you put it like that...